rules/prometheus-rules.yaml

```diff
@@ -1911,25 +2024,48 @@ spec:
         sum by (namespace, pod) (kube_pod_status_phase{job="kube-state-metrics", phase=~"Failed"}) > 0
       for: 1h
       labels:
+        page: "true"
         severity: critical
   - name: mintel-web-frontend.alerts
     rules:
     - alert: MintelWebServiceDown
       annotations:
         description: Service `{{$labels.service}}`is down.
         runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/MintelWebServiceDown.md
-      expr: count(mintel:web_frontend:check_up) by (job, namespace, service) == 0
+      expr: count(mintel:web_frontend:check_up == 1) by (job, namespace, service,
+        environment, label_app_mintel_com_owner) == 0
+      for: 3m
+      labels:
+        page: "true"
+        severity: critical
+    - alert: MintelWebServiceDown
+      annotations:
+        description: Service `{{$labels.service}}`is down.
+        runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/MintelWebServiceDown.md
+      expr: count(mintel:web_frontend:check_up == 1) by (job, namespace, service,
+        environment, label_app_mintel_com_owner) == 0
       for: 1m
       labels:
+        severity: warning
+    - alert: MintelReducedService
+      annotations:
+        description: Service `{{$labels.service}}`is down.
+        runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/MintelReducedService.md
+      expr: count(mintel:web_frontend:check_up == 1) by (job, namespace, service,
+        environment, label_app_mintel_com_owner) == 1
+      for: 3m
+      labels:
+        page: "true"
         severity: critical
     - alert: MintelReducedService
       annotations:
         description: Service `{{$labels.service}}`is down.
         runbook_url: https://gitlab.com/mintel/satoshi/docs/blob/master/runbooks/core/MintelReducedService.md
-      expr: count(mintel:web_frontend:check_up) by (job, namespace, service) == 1
+      expr: count(mintel:web_frontend:check_up == 1) by (job, namespace, service,
```

Comment:
Just a note - this doesn't work because the recording rule here is wrong. It still checks `vendor=mintel` which we don't set anymore.

Are you ok in us fixing that in a subsequent PR?

Reply:
i fixed this but i must have lost it in one of my stash or something. 

i will fix this here

Reply:
this is in the MR 

https://github.com/mintel/kube-monitoring-mixins/pull/26/files?file-filters%5B%5D=.libsonnet&file-filters%5B%5D=.yaml#diff-4e3cd1becc605f67d68d1eac35e7c5ddR11